thorlcr/graph/thgraphmaster.hpp

```diff
@@ -48,7 +48,9 @@ interface IJobManager : extends IInterface
     virtual void setWuid(const char *wuid, const char *cluster=NULL) = 0;
     virtual IDeMonServer *queryDeMonServer() = 0;
     virtual void fatal(IException *e) = 0;
+#ifndef _CONTAINERIZED
```

Comment:
I wonder if it is better to include this in the container build for a few reasons
- code is simpler, and compile errors are caught for both versions
- we might want to store dlls on storage that can't be mounted - then there would be a need to copy locally again.
- might we want to copy locally anyway to avoid re-read costs, and ensure it is read all in one go.  At least explore the relative performance it was written to a local nvme drive.
-

Reply:
If dlls are on storage that can't be mounted / it cost money to keep reading them from, then we would probably want to rethink Roxie behaviour too...

Reply:
Yes we probably should...